tasks/facial_landmark_detection/latency.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import io
+from typing import Dict, List, Optional, Tuple, Union
+
+import os
+os.environ["OMP_NUM_THREADS"] = "1"
+import statistics
+from time import perf_counter
+
+import onnxruntime as rt
+import torch
+
+from archai.discrete_search.api.archai_model import ArchaiModel
+
+
+class AvgOnnxLatency:
```

Comment:
Can this be merged with https://github.com/microsoft/archai/blob/4a3cf62a77ffef12390b9c1973fc22eda4e2359a/archai/discrete_search/evaluators/onnx_model.py ?

Reply:
Tough call on this one. Mine evolved from onnx_model.py for an explict reason. The way it is done there doesn't allow std deviation computation. However, the stdev computation is removed for simplicity in this example. So seeminly I can just use the existing one. That would change all my results however. :( If we take my code, everybody else's results will vary. I'm thinking that is too much distablization.